app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players_list: list[Player]) -> int:
+    return sum([player.get_rating() for player in players_list])
+
+
+def elves_concert(elf_list: list[Elf]) -> None:
+    for elf in elf_list:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarf_list: list[Dwarf]) -> None:
+    for dwarf in dwarf_list:
```

Comment:
dwarf in plural is `dwarves`

Reply:
Ok, thanks! And i miss elf in plural too. Now i will change.